queries.sql

```diff
@@ -22,4 +22,47 @@ SELECT * FROM animals WHERE neutered = 't';
 SELECT * FROM animals WHERE name NOT LIKE '%Gabumon%';
 
 /* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
-SELECT * FROM animals WHERE weight_kg >= '10.4' AND weight_kg <= '17.3';
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= '10.4' AND weight_kg <= '17.3';
+
```

Comment:
Please update the `animals` table by setting the species column to `unspecified`, then roll back the change and verify that change was made.